### PR TITLE
returns list of target altitudinal zones  as options

### PR DESCRIPTION
**Pull request by angsherpa232**

### CODE REVIEW

File: src/project.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ const altitudinalZoneList = types.altitudinalZone
 const getNextHeigtLevel = currentaltitudinalZone =>
   altitudinalZoneList[altitudinalZoneList.indexOf(currentaltitudinalZone) + 1];
 
-function projectionReducer(location) {
+function projectionReducer(location, targetaltitudinalZone) {
```
Comment (friedjoff) [inline src/project.js]:
```suggestion
function projectionReducer(location, targetAltitudinalZone) {
```

Please use camelCase.

File: src/project.js [hunk 2]
```diff
@@ -70,31 +70,51 @@ function projectionReducer(location) {
     newLocation.forestType = forestType;
   }
 
-  newLocation.altitudinalZone = getNextHeigtLevel(location.altitudinalZone);
+  if (location.altitudinalZone && targetaltitudinalZone !== undefined) {
+    newLocation.altitudinalZone = getNextHeigtLevel(location.altitudinalZone);
+  } else {
+    newLocation.altitudinalZone = location.altitudinalZone;
```
Comment (friedjoff) [inline src/project.js]: Is this necessary? I guess this should be coverd by line 44.

File: src/project.js [hunk 3]
```diff
@@ -70,31 +70,51 @@ function projectionReducer(location) {
     newLocation.forestType = forestType;
   }
 
-  newLocation.altitudinalZone = getNextHeigtLevel(location.altitudinalZone);
+  if (location.altitudinalZone && targetaltitudinalZone !== undefined) {
+    newLocation.altitudinalZone = getNextHeigtLevel(location.altitudinalZone);
```
Comment (friedjoff) [inline src/project.js]: Can you rename the function to getNextAltitudinalZone. BTW: adding a comment to the function about why next zone is lower might be a good idea :)

File: src/project.js [hunk 4]
```diff
@@ -70,31 +70,51 @@ function projectionReducer(location) {
...
-    newLocation = projectionReducer(location);
-    if (newLocation.altitudinalZone !== targetaltitudinalZone) {
+
+  if (
+    altitudinalZoneList[altitudinalZonePointer] !== targetaltitudinalZone ||
+    targetaltitudinalZone === undefined
+  ) {
+    newLocation = projectionReducer(location, targetaltitudinalZone);
+    if (
+      newLocation.altitudinalZone !== undefined &&
```
Comment (friedjoff) [inline src/project.js]: I guess we can remove line 104 because line 106 already checks if they are equal.

File: src/project.js (hunk 2 above)
Comment (angsherpa232) [reply src/project.js]: This solved the problem of tree-app not showing the options for altitudinalZone to choose from. The reason was that if the initial condition is not checked in the library then the options object from returned newLocation will only contain targetHeightLevel as "10" and no other options such as forestEcoregion etc.

File: src/project.js (hunk 3 above)
Comment (angsherpa232) [reply src/project.js]: Agree!

File: src/project.js (hunk 2 above)
Comment (angsherpa232) [reply src/project.js]: getNextHeightLevel (changed to getNextAltitudinalZone) moved to line 73 so above if statement is not required anymore.

File: src/project.js (hunk 4 above)
Comment (angsherpa232) [reply src/project.js]: otherwise the test breaks.

File: src/project.js [hunk 5]
```diff
@@ -37,12 +37,15 @@ const altitudinalZoneList = types.altitudinalZone
   .map(e => e.code)
   .reverse();
 
-const getNextHeigtLevel = currentaltitudinalZone =>
+/* Provides the list of altitudinal Zones as target Altitudinal zones that are immediately 
+after the currently chosen altitudinal Zone. */
```
Comment (friedjoff) [inline src/project.js]: Event better would by a description _why_ we do it 😉

**Pull request merged**